lib/rules/no-tryinvoke.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const types = require('../utils/types');
+const { getImportIdentifier } = require('../utils/import');
+
+const ERROR_MESSAGE = 'Use optional chaining operator `?.()` instead of `tryInvoke`';
+
+module.exports = {
+  ERROR_MESSAGE,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: "disallow usage of the Ember's `tryInvoke` util",
+      category: 'Ember Utils',
+      recommended: false,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-tryinvoke.md',
+    },
+    fixable: null,
```

Comment:
In a follow-up PR since it might be a bit tricky, we can probably make this rule autofixable